src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -57,4 +58,8 @@ def load_arguments(self, _):
         c.argument('azure_file_volume_mount_path', validator=validate_volume_mount_path, help='The path within the container where the volume should be mounted. Must not contain colon (:).')
 
     with self.argument_context('container logs') as c:
-        c.argument('container_name', help='The container name to tail the logs')
+        c.argument('container_name', help='The container name to tail the logs. If omitted, the first container in the container group will be chosen')
+        c.argument('follow', help='Indicate to stream the tailing logs', arg_type=get_three_state_flag(return_label=True))
```

Comment:
We usually reserve the use of `get_three_state_flag` for registering boolean properties. For switches on the command, we recommend the `action='store_true'` kwarg.